spec/factories/conferences.rb

```diff
@@ -54,5 +54,41 @@
         conference.save!
       end
     end
+
+    factory :semi_conference do
```

Comment:
This factory created a full_conference but with a non public splashpage. As a situation may occur, where only one conference is love but has no public splashpage. So to test this situation i made a semi _conference .